diff --git a/mediawiki/common.js b/mediawiki/common.js
index f0ef640916c6929b95dd311cc43ae0d252c7ec83..e794b9c14b5df99cc83123604f199145bb7aa4fb 100644
--- a/mediawiki/common.js
+++ b/mediawiki/common.js
@@ -38,10 +38,6 @@ const config = {
 						"component": "never"
 					}
 				} ],
-				"vue/component-name-in-template-casing": [ "error", "kebab-case" ],
-				"vue/no-child-content": [ "error", {
-					"additionalDirectives": [ "i18n-html" ]
-				} ]
 			}
 		}
 	]
