diff --git a/mediawiki/common.js b/mediawiki/common.js
index f0ef640..8fa0d04 100644
--- a/mediawiki/common.js
+++ b/mediawiki/common.js
@@ -37,10 +37,6 @@ const config = {
 						"normal": "never",
 						"component": "never"
 					}
-				} ],
-				"vue/component-name-in-template-casing": [ "error", "kebab-case" ],
-				"vue/no-child-content": [ "error", {
-					"additionalDirectives": [ "i18n-html" ]
 				} ]
 			}
 		}
