diff --git a/index.d.ts b/index.d.ts
index 9f325ead3f8908e6465e2391b61af6037af61c6b..bef01b1432d67777196d6eb3d7718850785e4887 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -51,6 +51,7 @@ export interface ApiParams extends UnknownApiParams {
     responselanginfo?: boolean;
     origin?: string;
     uselang?: string;
+    variant?: string;
     errorformat?: "bc" | "html" | "none" | "plaintext" | "raw" | "wikitext";
     errorlang?: string;
     errorsuselocal?: boolean;
@@ -1364,19 +1365,7 @@ export interface DiscussionToolsApiDiscussionToolsEditParams extends ApiParams {
     /**
      * Apply the selected skin to the parser output. May affect the following properties: `text`, `langlinks`, `headitems`, `modules`, `jsconfigvars`, `indicators`.
      */
-    useskin?:
-        | "apioutput"
-        | "authentication-popup"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "json"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     /**
      * Unconditionally add or remove the page from the current user's watchlist, use preferences (ignored for bot users) or do not change watch.
      */
@@ -1514,19 +1503,7 @@ export interface DiscussionToolsApiDiscussionToolsPreviewParams extends ApiParam
     /**
      * Apply the selected skin to the parser output. May affect the following properties: `text`, `langlinks`, `headitems`, `modules`, `jsconfigvars`, `indicators`.
      */
-    useskin?:
-        | "apioutput"
-        | "authentication-popup"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "json"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     /**
      * Return parse output in a format suitable for mobile devices.
      */
@@ -2869,17 +2846,7 @@ export interface ApiImportParams extends ApiParams {
     /**
      * For interwiki imports: wiki to import from.
      */
-    interwikisource?:
-        | "commons"
-        | "de"
-        | "es"
-        | "fr"
-        | "it"
-        | "meta"
-        | "nost"
-        | "outreachwiki"
-        | "pl"
-        | "test2wiki";
+    interwikisource?: string;
     /**
      * For interwiki imports: page to import.
      */
@@ -4224,19 +4191,7 @@ export interface ApiParseParams extends ApiParams {
     /**
      * Apply the selected skin to the parser output. May affect the following properties: `text`, `langlinks`, `headitems`, `modules`, `jsconfigvars`, `indicators`.
      */
-    useskin?:
-        | "apioutput"
-        | "authentication-popup"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "json"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     /**
      * Content serialization format used for the input text. Only valid when used with text.
      */
@@ -6538,19 +6493,7 @@ export interface VisualEditorApiVisualEditorEditParams extends ApiParams {
     /**
      * Apply the selected skin to the parser output. May affect the following properties: `text`, `langlinks`, `headitems`, `modules`, `jsconfigvars`, `indicators`.
      */
-    useskin?:
-        | "apioutput"
-        | "authentication-popup"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "json"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     /**
      * Change tags to apply to the edit.
      */
