diff --git a/index.d.ts b/index.d.ts
index 9e1ed1acf8d0e28e3e6919b2898d65bc9ded2c99..4dd6d5ab58b809b4051113c12f3ea55a69970621 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -51,6 +51,7 @@ export interface ApiParams extends UnknownApiParams {
     responselanginfo?: boolean;
     origin?: string;
     uselang?: string;
+    variant?: string;
     errorformat?: "bc" | "html" | "none" | "plaintext" | "raw" | "wikitext";
     errorlang?: string;
     errorsuselocal?: boolean;
@@ -438,17 +439,7 @@ export interface DiscussionToolsApiDiscussionToolsEditParams extends ApiParams {
     summary?: string;
     sectiontitle?: string;
     allownosectiontitle?: boolean;
-    useskin?:
-        | "apioutput"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     watchlist?: string;
     captchaid?: string;
     captchaword?: string;
@@ -482,17 +473,7 @@ export interface DiscussionToolsApiDiscussionToolsPreviewParams extends ApiParam
     page?: string;
     wikitext?: string;
     sectiontitle?: string;
-    useskin?:
-        | "apioutput"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     mobileformat?: boolean;
 }
 
@@ -915,17 +896,7 @@ export interface ApiImportParams extends ApiParams {
     summary?: string;
     xml?: upload;
     interwikiprefix?: string;
-    interwikisource?:
-        | "commons"
-        | "de"
-        | "es"
-        | "fr"
-        | "it"
-        | "meta"
-        | "nost"
-        | "outreachwiki"
-        | "pl"
-        | "test2wiki";
+    interwikisource?: string;
     interwikipage?: string;
     fullhistory?: boolean;
     templates?: boolean;
@@ -1357,17 +1328,7 @@ export interface ApiParseParams extends ApiParams {
     preview?: boolean;
     sectionpreview?: boolean;
     disabletoc?: boolean;
-    useskin?:
-        | "apioutput"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     contentformat?:
         | "application/json"
         | "application/octet-stream"
@@ -2764,17 +2725,7 @@ export interface VisualEditorApiVisualEditorEditParams extends ApiParams {
     returnto?: string;
     returntoquery?: string;
     returntoanchor?: string;
-    useskin?:
-        | "apioutput"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     tags?: string | string[];
     plugins?: string | string[];
     mobileformat?: boolean;
