diff --git a/index.d.ts b/index.d.ts
index 6a0d495..deeb1fe 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -51,6 +51,7 @@ export interface ApiParams extends UnknownApiParams {
     responselanginfo?: boolean;
     origin?: string;
     uselang?: string;
+    variant?: string;
     errorformat?: "bc" | "html" | "none" | "plaintext" | "raw" | "wikitext";
     errorlang?: string;
     errorsuselocal?: boolean;
@@ -473,19 +474,7 @@ export interface DiscussionToolsApiDiscussionToolsEditParams extends ApiParams {
     summary?: string;
     sectiontitle?: string;
     allownosectiontitle?: boolean;
-    useskin?:
-        | "apioutput"
-        | "authentication-popup"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "json"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     watchlist?: string;
     captchaid?: string;
     captchaword?: string;
@@ -519,19 +508,7 @@ export interface DiscussionToolsApiDiscussionToolsPreviewParams extends ApiParam
     page?: string;
     wikitext?: string;
     sectiontitle?: string;
-    useskin?:
-        | "apioutput"
-        | "authentication-popup"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "json"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     mobileformat?: boolean;
 }

@@ -982,17 +959,7 @@ export interface ApiImportParams extends ApiParams {
     summary?: string;
     xml?: upload;
     interwikiprefix?: string;
-    interwikisource?:
-        | "commons"
-        | "de"
-        | "es"
-        | "fr"
-        | "it"
-        | "meta"
-        | "nost"
-        | "outreachwiki"
-        | "pl"
-        | "test2wiki";
+    interwikisource?: string;
     interwikipage?: string;
     fullhistory?: boolean;
     templates?: boolean;
@@ -1435,19 +1402,7 @@ export interface ApiParseParams extends ApiParams {
     preview?: boolean;
     sectionpreview?: boolean;
     disabletoc?: boolean;
-    useskin?:
-        | "apioutput"
-        | "authentication-popup"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "json"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     contentformat?:
         | "application/json"
         | "application/octet-stream"
@@ -2895,19 +2850,7 @@ export interface VisualEditorApiVisualEditorEditParams extends ApiParams {
     returnto?: string;
     returntoquery?: string;
     returntoanchor?: string;
-    useskin?:
-        | "apioutput"
-        | "authentication-popup"
-        | "cologneblue"
-        | "contenttranslation"
-        | "fallback"
-        | "json"
-        | "minerva"
-        | "modern"
-        | "monobook"
-        | "timeless"
-        | "vector"
-        | "vector-2022";
+    useskin?: string;
     tags?: string | string[];
     plugins?: string | string[];
     mobileformat?: boolean;
